recipes-iec61850/iec61850-open-server/iec61850-open-server.bb

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2021, Alliander (http://www.alliander.com)
+# SPDX-License-Identifier: Apache-2.0
+
+LICENSE = "GPL-3.0"
+LIC_FILES_CHKSUM = "file://COPYING;md5=d32239bcb673463ab874e80d47fae504"
+
+SRC_URI = "git://github.com/robidev/iec61850_open_server.git;protocol=https;branch=master"
+
+#PV = "1.0+git${SRCPV}" # TODO check if this should be included
+SRCREV = "${AUTOREV}"
+DEPENDS = "libiec61850"
+RDEPENDS_${PN} = "libiec61850" 
+
+S = "${WORKDIR}/git"
+
+inherit cmake
+
+# NOTE: this project uses a dual license scheme, as libiec61850 is covered by GPL 3.0
+# this means that any software that incorporates this library should adhere to GPL 3.0 as well
+# however, the circuit-simulator does not use libiec61850, and therefore this is apache licensed
+# the circuit-simulator should normally be run on a separate, non-realtime test machine, and is 
+# therefore ommitted from the recipe
+
+# TODO see if/why this cannot be done from cmake config directly
+FILES_${PN} = "/opt/iec61850_open_server/" 
```

Comment:
yes you should handle that with cmake, I believe.

Reply:
any help on how? I was not able to prevent the warning, even after creating the rules for packaging in cmake, it still seems to provide this warning.

Reply:
Unfortunately, the variable is hard-coded in the CMakeLists.txt.
https://github.com/robidev/iec61850_open_server/blob/85127b1028c0b41108bb765c9585f7c2eb00587a/CMakeLists.txt#L10
`
SET(CMAKE_INSTALL_PREFIX /opt)`

Reply:
Thanks! To be honest, this was my first attempt at a cmake-file, so this helps me learn. I will update it

Reply:
Should be fixed now, have removed the CMAKE_INSTALL_PREFIX from the makefile, and let the recipe decide where to install (but usr/bin does not work due to the way config files are processed/read, so it still deploys to /opt)